recipes/userver/all/conanfile.py

```diff
@@ -0,0 +1,287 @@
+
+required_conan_version = '>=2.8.0'  # pylint: disable=invalid-name
+
+
+class UserverConan(ConanFile):
+    name = 'userver'
+    description = 'The C++ Asynchronous Framework'
+    topics = ('framework', 'coroutines', 'asynchronous')
+    url = 'https://github.com/conan-io/conan-center-index'
+    homepage = 'https://userver.tech/'
+    license = 'Apache-2.0'
+    package_type = 'static-library'
+
+    settings = 'os', 'arch', 'compiler', 'build_type'
+    options = {
+        'fPIC': [True, False],
+        'lto': [True, False],
+        'with_jemalloc': [True, False],
+        'with_mongodb': [True, False],
+        'with_postgresql': [True, False],
+        'with_postgresql_extra': [True, False],
+        'with_redis': [True, False],
+        'with_redis_tls': [True, False],
+        'with_grpc': [True, False],
+        'with_clickhouse': [True, False],
+        'with_rabbitmq': [True, False],
+        'with_utest': [True, False],
+        'with_kafka': [True, False],
+        'with_otlp': [True, False],
+        'with_sqlite': [True, False],
+        'with_easy': [True, False],
+        'with_s3api': [True, False],
+        'with_grpc_reflection': [True, False],
+        'with_grpc_protovalidate': [True, False],
+        'namespace': ['ANY'],
+        'namespace_begin': ['ANY'],
+        'namespace_end': ['ANY'],
+        'python_path': ['ANY'],
+    }
+
+    default_options = {
+        'fPIC': True,
+        'lto': False,
+        'with_jemalloc': (platform.system() != 'Darwin'),
+        'with_mongodb': True,
+        'with_postgresql': True,
+        'with_postgresql_extra': False,
+        'with_redis': True,
+        'with_redis_tls': True,
+        'with_grpc': True,
+        'with_clickhouse': True,
+        'with_rabbitmq': True,
+        'with_utest': True,
+        'with_kafka': True,
+        'with_otlp': True,
+        'with_sqlite': True,
+        'with_easy': True,
+        'with_s3api': True,
+        'with_grpc_reflection': True,
+        'with_grpc_protovalidate': False,
+        'namespace': 'userver',
+        'namespace_begin': 'namespace userver {',
+        'namespace_end': '}',
+        'python_path': 'python3',
+        'mongo-c-driver/*:with_sasl': 'cyrus',
+        'grpc/*:php_plugin': False,
+        'grpc/*:node_plugin': False,
+        'grpc/*:ruby_plugin': False,
+        'grpc/*:csharp_plugin': False,
+        'grpc/*:objective_c_plugin': False,
+        'hiredis/*:with_ssl': True,
+        'librdkafka/*:ssl': True,
+        'librdkafka/*:curl': True,
+        'librdkafka/*:sasl': True,
+        'librdkafka/*:zlib': True,
+        'librdkafka/*:zstd': True,
+        're2/*:with_icu': True,
+    }
+
+    def source(self):
+        known_version = (self.conan_data or {}).get("sources", {}).get(self.version)
+        if known_version:
+            get(self, **known_version, strip_root=True)
+        else:
+            Git(self).clone('https://github.com/userver-framework/userver.git', target='.')
```

Comment:
in Conan Center we use https:// sources, rather than git - try to stick to the known pattern

Reply:
Just removed that branch - looks like we do not need it at all